src/main/resources/META-INF/hudson.remoting.ClassFilter

```diff
@@ -1 +1,2 @@
+# For backward compatibility with 1.4.2-
 java.util.RandomAccessSubList
```

Comment:
Commenting out this line and running test with Jenkins 2.102 results following error:
```
testJEP200(jp.ikedam.jenkins.plugins.extensible_choice_parameter.IntegrationTest)  Time elapsed: 1.793 s  <<< FAILURE!
java.lang.AssertionError: expected:<ChoiceWithSublist1> but was:<null>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at jp.ikedam.jenkins.plugins.extensible_choice_parameter.IntegrationTest.testJEP200(IntegrationTest.java:52)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.jvnet.hudson.test.JenkinsRule$1.evaluate(JenkinsRule.java:542)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:745)
```

with following log:
```
   0.410 [id=1899]	WARNING	jenkins.security.ClassFilterImpl#lambda$isBlacklisted$1: java.util.RandomAccessSubList in JRE might be dangerous, so rejecting; see https://jenkins.io/redirect/class-filter/
```